ecs/README.md

```diff
@@ -133,32 +186,13 @@ INSTANCE_ID=$(aws ecs describe-container-instances --cluster l5d-demo --containe
 ECS_NODE=$(aws ec2 describe-instances --instance-ids $INSTANCE_ID --query Reservations[*].Instances[0].PublicDnsName --output text)
 
 # view linkerd-viz (osx)
-open http://$ECS_NODE:3000
+open http://localhost:3000
```

Comment:
above this line, add: `ssh -i "~/.ssh/$KEY_PAIR.pem" -L 127.0.0.1:3000:$VIZ_NODE:3000 ec2-user@$VIZ_NODE -N` ... and above that line, change `ECS_NODE=...` to `VIZ_NODE...`